cinema/serializers.py

```diff
@@ -51,7 +70,15 @@
 
     class Meta:
         model = Movie
-        fields = ("id", "title", "description", "duration", "genres", "actors")
+        fields = (
+            "id",
+            "title",
+            "description",
+            "duration",
+            "genres",
+            "actors",
+            "image",
```

Comment:
The `MovieDetailSerializer` correctly includes the `image` field, ensuring that the image URL is displayed in the movie detail view.